### PR TITLE
fix: removing bootstrap and updating tokens / neptune css documentation

### PR DESCRIPTION
**Pull request by helendurranttw**
## 🖼 Context

* Removed Bootstrap as a dependency
* Updated docs to reflect our new recommendations to use Neptune CSS
* Updated style lint rules to point to tokens

## ✅ Checklist

- [x] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: packages/css/docs/upgrading.md [hunk 1]
```diff
@@ -6,10 +6,10 @@
 
 - The full Neptune CSS bundle that contains `neptune-tokens`, `neptune-core`, `neptune-addons`, and all available CSS components.
 
-### `neptune-tokens.css`
+### Variables
 
-- Semantic design tokens are now available as custom properties (CSS variables).
-- Variables are still available as partials from `dist/css/variables` but usage of tokens from `neptune-tokens.css` is preferred.
+- Semantic design tokens are now available as custom properties (CSS variables) from the @transferwise/neptune-tokens repo.
+- Variables are still available from legacy-variables.less, but use of custom properties is encouraged.
```
Comment (dozortsev) [inline packages/css/docs/upgrading.md:12]: > ..Variables are still available from legacy-variables.less...

I would be more precise here saying

> ..Bootstrap variables are still available from legacy-variables.less..

File: packages/docs/liveEditorCode/accordion.code.js [hunk 2]
```diff
@@ -6,23 +6,12 @@
...
-    {
-      title: 'Item 4',
-      content: (
-        <ul>
-          <li>This</li>
-          <li>is</li>
-          <li>a</li>
-          <li>list</li>
-        </ul>
-      ),
```
Comment (dozortsev) [inline packages/docs/liveEditorCode/accordion.code.js:25]: this example was demoing ability to pass there `node` why deleting it?

Comment (helendurranttw) [reply packages/docs/liveEditorCode/accordion.code.js:25]: I'll update with a new node example.

**Comment by helendurranttw** [comment]
> have you checked docs for visual bugs after migration on `neptune.css`? 🙂

I didn't remove neptune.css in this commit, that bug must be in dev 😬.. I'll investigate.

File: packages/docs/pages/about/Setup.mdx [hunk 3]
```diff
@@ -55,73 +39,53 @@ export default function Hello() {
...
 
-Browse our <DocLink href="/components">components</DocLink> for usage documentation, live previews and code snippets for each component.
+For information on what classes are available, see the <DocLink href="/styles/core/Typography">styles documentation</DocLink>. Refer to <DocLink href="/components/inputs/Button">components</DocLink> for usage documentation, live previews and code snippets for each component.
+
+By default, apps come polyfilled. If you would like to use an unpolyfilled version because your app provides its own polyfills, see [Advanced Components Usage](#advanced-components-usage) below.
 
-### Installing CSS
+### Tokens
 
+Neptune CSS exposes a number of of useful CSS properties on `:root`. You can find information about what is available in the <DocLink href="/components/tokens">tokens documentation</DocLink>. If you're not familiar with using custom properties, have a read of the documentation on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
```
Comment (andreapiras) [inline packages/docs/pages/about/Setup.mdx:48]: of of

**Pull request merged**